(doc) Add user model and post model

MauriceOkumu opened the pull request:
Add user model and post model

MauriceOkumu committed:
(doc) Add user model and post model

anthonypecchillo commented on db/models/postModel.js:
```diff
@@ -0,0 +1,16 @@
+  },
+  content: {
+  	type:Sequelize.TEXT,
+  	allowNull: false,
```
Think you might have an extra comma inside the object literal, otherwise, this request is ready to be merged!  Nice work Maurice.

anthonypecchillo replied:
And put a space after 'type' on line 11.

anthonypecchillo commented on db/models/userModel.js:
```diff
@@ -0,0 +1,28 @@
+  cohort: Sequelize.STRING,
+  status: Sequelize.STRING,
+  github: {
+  	type:Sequelize.STRING,
```
Put a space after 'type' on line 17.

anthonypecchillo commented on db/models/userModel.js:
```diff
@@ -0,0 +1,28 @@
+  	allowNull: false
+  },
+  email: {
+  	type:Sequelize.STRING,
```
Put a space after 'type' on line 11.

anthonypecchillo replied:
Is it okay for line 7 to be:  'Sequelize.STRING'

Rather than:  'type: Sequelize.STRING'  ?

anthonypecchillo reviewed (changes requested):
I noticed a few small typos (1 necessary, 3 stylistic) that should be fixed before merging.  I left a comment on each line.

anthonypecchillo committed:
Update postModel.js

- Added space after 'type:' on line 11.
- Removed comma after 'allowNull: false' on line 12.

anthonypecchillo committed:
(fix) Fix minor typos.

- Add property 'type' to line 7
- Add space after 'type:' on line 11
- Add space after 'type:' on line 17
